app/frontend/src/components/Welcome.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <v-container>
+    <v-row class>
```

Comment:
Should try and set this up without nested grids, as nothing floats next to each other.

Reply:
thanks. please see updated code